index.js

```diff
@@ -84,10 +84,18 @@ module.exports = (input, opts) => {
 		});
 	}
 
-	tasks.add({
-		title: 'Bumping version',
-		task: () => exec('npm', ['version', input])
-	});
+	tasks.add([
+		{
+			title: 'Bumping version using Yarn',
```

Comment:
@SamVerschueren I considered that too, but I think separate tasks are fine. That also makes it easier to add other Yarn specific tweaks in the future.